04_numeronym/madhavansu/main_test.go

```diff
@@ -0,0 +1,46 @@
+package main
+
+import (
+	"bytes"
+	"fmt"
+	"strconv"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+var s = []struct {
+	input  string
+	output string
+}{
+	{"accessibility", "[a11y]"},
+	{"Kubernetes", "[K8s]"},
+	{"World Wide Web", "[W12b]"},
+	{"abc", "[abc]"},
+}
+
+func TestMainOutput(t *testing.T) {
+	// Given
+	r := require.New(t)
+	var buf bytes.Buffer
+	out = &buf
+
+	// When
+	main()
+
+	// Then
+	expected := strconv.Quote(`[a11y K8s abc]`)
```

Comment:
you probably dont need to wrap both `expected` and `actual` in `strconv.Quote`. Compare them directly.

Reply:
This is for the readability purpose, as a user we will know what is expected and what is the actual value.

Reply:
This is really a personal preference thing. If they are complicated to construct we usually prefer `expected` and `actual`, otherwise its really up to the programmer.